sandbox/src/timebar/timebar.js

```diff
@@ -147,13 +171,19 @@ class TimebarContainer extends Component {
           onChange={this.update}
           onMouseMove={this.onMouseMove}
           onBookmarkChange={this.updateBookmark}
+          // minimumRange={1}
```

Comment:
@nerik remove commented code, or of it has any usefulness add a comment above the commented lines stating what they are for:

```
// enable lines below for testing X
```